jobs/JRRFS_UNGRIB

```diff
@@ -18,18 +32,35 @@ else
 fi
 export jobid=${jobid:-${RUN}_ungrib_${cyc}}
 export DATA=${DATA:-${DATAROOT}/${jobid}}
-export DATA=${DATA}/${FHR:-"FHR_not_defined"}
+#export DATA=${DATA}/${FHR:-"FHR_not_defined"}
 
 if [ -d ${DATA} ]; then # remove the ${DATA} directory if existed
   rm -rf ${DATA}
 fi
 mkdir -p ${DATA}
 #
-export COMINgfs=${COMINgfs:-$(compath.py $envir/gfs/$gfs_ver)}
-export COMINgefs=${COMINgefs:-$(compath.py $envir/gefs/$gefs_ver)}
-export COMINrap=${COMINrap:-$(compath.py $envir/rap/$rap_ver)}
-export COMINrrfs=${COMINrrfs:-$(compath.py $envir/rrfs/$rrfs_ver)}
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${rrfs_ver}/${RUN}.${PDY}/${cyc})}
+#-----------------------------------------------------------------------
+# Set the parameters for the external model files.
+#-----------------------------------------------------------------------
+#
+#export COMINgfs=${COMINgfs:-$(compath.py $envir/gfs/$gfs_ver)}
+#export COMINgefs=${COMINgefs:-$(compath.py $envir/gefs/$gefs_ver)}
+export COMIN=${SOURCE_BASEDIR:-("")}
+export NAME_PATTERN=${NAME_PATTERN:-("")}
+export OFFSET=${OFFSET:-0}
+export LENGTH=${LENGTH:-0}
+export INTERVAL=${INTERVAL:-1}
+#
+#-----------------------------------------------------------------------
+# Define COM directories
+#-----------------------------------------------------------------------
+#
+if [[ -d "${COMROOT}" ]];then
+  COMOUT=${COMROOT}/${NET}/${rrfs_ver}/${RUN}.${PDY}/${cyc}
+else
+  COMOUT=$(compath.py -o ${NET}/${rrfs_ver}/${RUN}.${PDY}/${cyc})
+fi
```

Comment:
I would think 
`export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${rrfs_ver}/${RUN}.${PDY}/${cyc})}`
works better as it is used/recommended by the NCO implementation standard.

Reply:
The COMOUT is decided by definition of COMROOT. Your suggestion only works on operational machine.

Reply:
> The COMOUT is decided by definition of COMROOT. Your suggestion only works on operational machine.

It works on both NCO and non-NCO. 
For non-NCO, one only needs to define the COMOUT variable in the launch.sh

Reply:
added COMOUT in xml and use:
export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${rrfs_ver}/${RUN}.${PDY}/${cyc})}
in J-job.